mapbook-iOS/AppSettings.swift

```diff
@@ -68,7 +68,10 @@ struct AppSettings {
     // When a user attempts to access a Portal, this URL string defaults in `PortalAccessViewController`
     // Change the cooresponding value in Info.plist to configure your own default
     // Falls back to arcgis online
-    static let defaultPortalURLString: String = getAgsSetting(named: "DefaultPortalURLString") ?? "https://www.arcgis.com"
+    static let defaultPortalURLString: String = getAgsSetting(named: "DefaultPortalURLString") ?? arcGISOnlineURLString
+    
+    // URL String to ArcGIS Online
+    static let arcGISOnlineURLString: String = "https://www.arcgis.com"
```

Comment:
I would suggest adding constants like this on the related type. Since this is a URL, you could declare it like this:

```swift
extension URL {
    static let arcGISOnline = URL(string: "https://www.arcgis.com")!
}
```

And then use it as follows:

```swift
static let defaultPortalURLString: String = getAgsSetting(named: "DefaultPortalURLString") ?? URL.arcGISOnline.absoluteString
```